src/components/Q&A/AnswerReport.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import {API_KEY} from '../../config/config.js';
+const axios = require('axios');
+
+class AnswerReport extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      currentAns: this.props.ansObj
+    }
+    this.handleReport = this.handleReport.bind(this);
+  }
+
+  handleReport(e) {
```

Comment:
quick reminder to auto fix the indentation (opt + shift + F) and rename any temp vars